model/shared/src/main/scala/explore/model/boopickle/package.scala

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+package object boopickle extends BoopicklePlatform {}
```

Comment:
Why do we need this an the empty `BoopicklePlatform`?

Reply:
I want to have a few facilities that are js only. To convert binary blobs to Transferrable objects

Reply:
I think we could move it